source/index.html.md

```diff
@@ -36,16 +51,36 @@ The [official documentation](https://www.stellar.org/developers/guides/anchor/in
 
 In this tutorial you will use Stellar to build a low-cost financial service similar to Venmo and instead of following the approach number two which is already documented in the Stellar website, you'll be maintaining a Stellar account for each customer and also making it transparant to the final user that they are using Stellar.
 
+
+[MOVE: This should go higher than the anchor definition. I'd make this part of the introduction]
```

Comment:
I figured there is a lot of redundancy in that paragraph too, I made it simpler:
```
The [official documentation](https://www.stellar.org/developers/guides/anchor/index.html#customer-accounts) covers the second method but there is no documentation about the first one. This tutorial will show you how to create an anchor maintaining a Stellar account for each customer.
```